ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot

```diff
@@ -58,6 +65,37 @@ Test Onnx Model Rest Inference Via UI (Triton on Kserve)
     ...  AND
     ...  Clean All Models Of Current User
 
+Test PYTORCH Model Inference Via UI(Triton on Kserve)
+    [Documentation]    Test the deployment of an pytorch model in Kserve using Triton
+    [Tags]    Sanity           RHOAIENG-11561
+
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
```

Comment:
this will fail because it will try to create the same project which got created in the previous test of the same suite and you're using the "existing_project" param se to TRUE